src/missing_prices.py

```diff
@@ -112,35 +117,29 @@ def fetch_tokens_without_prices(dune: DuneAPI) -> list[CoinPaprikaToken]:
     return [CoinPaprikaToken.from_dict(r) for r in results]
 
 
-def run_missing_prices():
+def run_missing_prices() -> None:
     """Script's Main Entry Point"""
     print("Getting Coin Paprika token list")
     coins = load_coins()
     print(f"Loaded {len(coins)} coins from Coin Paprika")
 
-    # Fetch tokens orders by descending, popularity
-    print("Getting traded tokens without prices from: https://dune.com/queries/224239")
-    tokens = sorted(
-        # Could be interesting to just get the results without executing.
-        fetch_tokens_without_prices(dune=DuneAPI.new_from_environment()),
-        key=lambda t: t.popularity,
-        reverse=True,
-    )
+    tokens = load_tokens(DuneClient(api_key=os.environ["DUNE_API_KEY"]))
     print(f"Fetched {len(tokens)} traded tokens from Dune without prices")
     found, res = 0, []
     for token in tokens:
-        if token.symbol in coins:
-            possibilities = coins[token.symbol]
-            if len(possibilities) == 1:
-                res.append(token.as_dune_repr(possibilities[0]["id"]))
-                found += 1
-            else:
-                print(f"non unique {token}: {len(possibilities)} occurrences")
-        if found > 50:
-            print("Stopped at 50 results.")
-            break
-
-    write_results(results=res, path="./out", filename="missing-prices.yaml")
+        if token["address"].lower() in coins:
+            paprika_data = coins[token["address"].lower()]
+            dune_row = (
+                str(paprika_data["id"]),
+                "ethereum",
+                str(paprika_data["symbol"]),
+                str(paprika_data["address"].lower()),
+                int(token["decimals"]),
+            )
+            res.append(dune_row)
```

Comment:
Also for the future we might wanna look into https://marshmallow.readthedocs.io/en/stable/ for easier object serialization/deserialization . What's your take on that?

Reply:
Yes I would very much like to do that.

Reply:
I am just not so sure how well it will jive with the Ethereum Types. I have been having quite some difficulty with that lately.

Reply:
Awesome! We should definitely give it a go and see how we can work with it.

Reply:
I will create an issue for this and link it to the TODO about removing Any above.

Reply:
https://github.com/cowprotocol/data-misc/issues/20